journal/views/category_view.py

```diff
@@ -0,0 +1,43 @@
+import csv
+import os
+from django.http import JsonResponse
+from django.http import HttpResponseNotFound, HttpResponseBadRequest
+from django.views.generic import View
+
+_CATEGORY_TYPES = ['cas', 'learning_objectives']
+_CSV_FILE_DIR = os.path.join(os.getcwd(), 'journal', 'files')
+
+class CategoryJSONView(View):
+
+    def _get_categories_from_file(self, category):
+        """
+        Outputs a dictionary of the particular category type. CSV files
+        are expected to have the following format:
+
+        1,Creativity
+        2,Action
+        3,Service
+        """
+        category = str(category)
+        assert category in _CATEGORY_TYPES, 'Category must be in approved \
+                                            category types!'
+        types = {}
+        file_name = os.path.join(_CSV_FILE_DIR, '{0}.csv'.format(category))
+        with open(file_name) as csvfile:
+            reader = csv.reader(csvfile)
+            for row in reader:
+                types[row[0]] = row[1]
```

Comment:
row[0] and row[1] should be swapped out with variables that better convey their meaning.